app/src/main/res/layout/fragment_profile.xml

```diff
@@ -64,18 +68,18 @@
         app:layout_constraintTop_toBottomOf="@id/textbadges"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintEnd_toEndOf="parent"
-        android:layout_marginTop="20dp" />
+        android:layout_marginTop="20dp" />-->
 
     <include
         android:id="@+id/card_impact_layout"
         layout="@layout/card_impact"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
-        app:layout_constraintTop_toBottomOf="@id/textMyEvents"
-        app:layout_constraintStart_toStartOf="parent"
+        android:layout_marginTop="52dp"
         app:layout_constraintEnd_toEndOf="parent"
-        android:layout_marginTop="20dp"
-        android:layout_marginBottom="40dp"/>
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/txteditprofile" />
 
     <androidx.appcompat.widget.AppCompatButton
```

Comment:
nit: width could be wrap_content

Reply:
That will make the width squeezed!

Reply:
I tested the changes and it displays the same plus more dynamic